src/tests/ftest/util/server_utils.py

```diff
@@ -205,15 +205,6 @@ def prepare(self, storage=True):
         # Clean up any files that exist on the hosts
         self.clean_files()
 
-        if storage:
```

Comment:
We should also make these corresponding edits in https://github.com/daos-stack/daos/blob/4773c0df24383f8c0621c2c6b009bd7c4cf8e488/src/tests/ftest/util/server_utils.py#L174-L180:
```
    def prepare(self):
        """Prepare to start daos_server."""
```

And in https://github.com/daos-stack/daos/blob/master/src/tests/ftest/control/daos_admin_privileged.py#L55:
```
        self.server_managers[0].prepare()
```